### ISSUE TITLE
Define categories of examples

### ISSUE THREAD

**Issue by jezdez** [pyscript/pyscript#195]
While ramping up to a fully fledged documentation with a possible community example gallery, it would be valuable to set different categories of example we'd like to showcase.

**Comment by marimeireles** [comment]
Yes, this has been a topic of discussion both in the pyscript-collective part and with the folks who created the `py-config`.
I wonder what's the end of this discussion? @madhur-tandon do we currently have the possibility to tag examples using `config`?
Thanks!

**Comment by pauleveritt** [comment]
Yep, there's an effort in the "collective" to have a unified concept for contributed examples that ship separately from PyScript. I've done a first prototype of this. Currently stuck hoping to get some review by others, but perhaps I should just ship it.

**Comment by marimeireles** [comment]
Hey @pauleveritt what exactly are you talking about?
I thought I had reviewed everything you put out there?
Or are you looking for more folk's reviews?

**Comment by pauleveritt** [comment]
Yes, I was holding out for a technical review of the "Diary Driven Development" for any co-maintainers. But it's probably just too early. I should skip the DDD and just ship it.

**Comment by marimeireles** [comment]
You mean https://pscdemo.readthedocs.io/en/latest/building/proper_package.html correct?
I wish we had time to discuss a bit more on it, but personally I'd say go ahead. I think @fpliger and I gave some feedback here: https://github.com/pyscript/pyscript-collective/issues/25
But I think the general approach is that you should go forward with it...
I don't see any blockers from Fabio's and I part.
@tedpatrick approaches is also to give you a lot of freedom to move forward.
Please feel free to reach out for help.

**Comment by madhur-tandon** [comment]
> do we currently have the possibility to tag examples using `config`?

Not as of now unfortunately, CC: @marimeireles
